### PR TITLE
chore: add stacks support

### PR DESCRIPTION
**Pull request by chrisbbreuer**
The folks at Stacks are in full support of Ukraine 🙏🏼

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,5 +75,6 @@ This repository contains **Readme Banners** (and some useful docs) that can be u
 - [Sorry Cypress](https://github.com/sorry-cypress/sorry-cypress) - Open source alternative to Cypress Dashboard
 - [ProxyManager](https://github.com/Ocramius/ProxyManager) -  A  PHP library that aims to provide abstraction for generating various kinds of proxy classes
 - [FAR.js](https://github.com/farjs/farjs) - Cross-platform File and ARchive Manager in your terminal
+- [Stacks.js](https://github.com/stacksjs/stacks) - Develop modern clouds, apps & framework-agnostic libraries, faster
```
Comment (webknjaz) [inline README.md:78]: This should probably link the page where the banner appears:
```suggestion
- [Stacks.js](https://github.com/stacksjs) - Develop modern clouds, apps & framework-agnostic libraries, faster
```

**Review by webknjaz — APPROVED** [review]
Thanks 🙏

**Comment by webknjaz** [comment]
@chrisbbreuer FYI the website link linked from your org and repo seems to be broken. I suppose the domain DNS records are misconfigured.

**Comment by chrisbbreuer** [comment]
Thanks @webknjaz! I am actually currently "unbreaking" it and preparing it for a more so stable release, finally haha. One that is fully managed by the Stacks framework itself (code and cloud). Luckily, almost there! Haha

**Pull request merged**